AnnoDesigner/Localization/LocalizeBinding.cs

```diff
@@ -22,10 +22,20 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 #endif
         }
 
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+        {
+            return Convert(values[0], targetType, values[1], culture);
```

Comment:
Might want to check that `values.Length == 1`, `parameters.Length >= 2` (or whatever range check is required), and throw in Debug (so that we get a nice error message, rather than an `IndexOutOfRangeException`) and return when in release.

Reply:
I'll throw even in Release, WPF will handle it.